### ISSUE TITLE
[WCAG] Update Color Contrast Issues

### ISSUE THREAD

**Issue by arielkidwell** [vigetlabs/pointless-corp-astro#27]
**Documentation from Storyblok relaunch QA:**

https://pointlesscorp.netlify.app/storyboard/
- Top passes, bottom doesn't
- Have to go way darker to pass
![Image](https://github.com/user-attachments/assets/c48042b0-a14b-4cfa-9fa2-e9599b54b998)

https://pointlesscorp.netlify.app/baby-bookie/
- Have to go way darker to pass AND it conflicts with green logo graphic

https://pointlesscorp.netlify.app/starbnb/
- Had to go slightly lighter on link color which passes but makes it out of sync with. logo

https://pointlesscorp.netlify.app/gifbowl/
- Tried darkening background but then darker text got flagged

https://pointlesscorp.netlify.app/boom-carded/
- Accessibility nightmare

https://pointlesscorp.netlify.app/tracktor/
- Needs color rethink, if I pull one lever, another two break

https://pointlesscorp.netlify.app/jinglebots/
- Needs color rethink, if I pull one lever, another two break

https://pointlesscorp.netlify.app/scurry/
- Darker background works pretty well but needs updated logo

https://pointlesscorp.netlify.app/glimpsee/
- The pinkish text still doesn't meet it but not sure what other direction to go but white

https://pointlesscorp.netlify.app/the-data-driven-gardener/
- Needs color rethink, if I pull one lever, another two break (lots of green levels)
